_docs/manage/volumes/index.md

```diff
@@ -129,3 +129,22 @@ default/volume-name
 
 Volumes may not be removed immediately as the data will be purged in the
 background.
+
+## Using node selector
+
+Node selector is a form of constraint. For the volume to be eligible for a deployment on a node, selector must match node's labels. An empty selector
+always matches a node.
+
+If a group of nodes have a label `country=US`, then a node selector could be used
+during volume creation:
+
+```bash
+$ storageos volume create --nodeSelector 'country=US'
+```
+
+Another options for this case:
+
+```
+country in (US, CA)
+country notin (UK)
```

Comment:
If this syntax the same as K8s selectors, please link to https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors.